test/functional/apk-utils-e2e-specs.js

```diff
@@ -29,7 +29,7 @@ describe('apk utils', function () {
 
   before(async function () {
     adb = await ADB.createADB({
-      adbExecTimeout: (process.env.TRAVIS || process.env.CI) ? 60000 : 40000,
+      adbExecTimeout: (process.env.TRAVIS || process.env.CI) ? MOCHA_LONG_TIMEOUT : 40000,
```

Comment:
Also, I was expecting this (along with lines 82 + 93) to bump the timeout to the Travis long version of the timeout - e.g. 240000, am I setting it up wrong? (still seems to be timing out at https://travis-ci.org/appium/appium-adb/jobs/619672867?utm_medium=notification&utm_source=github_status )

Reply:
`waitDuration`, if set, has priority over the generic `adbExecTimeout` setting
See the implementation of `apkUtilsMethods.startApp` for more details

Reply:
do we still need to increase the timeout since we set `waitDuration` ?

Reply:
Just pushed a commit with this reverted; hopefully not needed any more :)